server/pulp/server/db/manage.py

```diff
@@ -191,6 +196,35 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        active_workers = status.get_workers()
+        if active_workers:
+            last_worker_time = max([worker['last_heartbeat'] for worker
+                                    in active_workers])
```

Comment:
Can this be joined with the list comprehension on the previous line? I think it would be < 100 chars. If it isn't, then leave it as is.